src/Uno.UWP/Devices/Geolocation/Geolocator.cs

```diff
@@ -166,7 +166,7 @@ private static void BroadcastStatusChanged(PositionStatus positionStatus)
 
 		private void StartStatusChanged() => _statusChangedSubscriptions.TryAdd(this, 0);
 
-		private void StopStatusChanged() => _statusChangedSubscriptions.TryRemove(this, out var _);
+		private void StopStatusChanged() => _statusChangedSubscriptions?.TryRemove(this, out var _);
```

Comment:
I don't quite see how this is correct. `_statusChangedSubscriptions` is supposed to never be null as it is initialized inline and is never set again to anything else.

Reply:
Maybe it is some race condition, with two calls - one is disposing object, and second is using it.
It is happening, in emulator, only when there is no valid geoposition, and doesn't happen when GetGeopositionAsync(cachetime) is called less than cachetime after this crash.
On my sister phone seems that it crashes every time.